nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,78 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.listen(3000, () => console.log(`Calculator:listening on port 3000`));
```

Comment:
Generally, you want this statement to be the very last thing in your file, so that the sever is fully configured before you start taking in requests.